src/scripts/bridging/getBungeeBridgeStatus.ts

```diff
@@ -0,0 +1,124 @@
+import { OrderBookApi, SupportedChainId } from '@cowprotocol/cow-sdk';
+import axios from 'axios';
+
+type BridgeStatusResponse = {
+  success: boolean;
+  result: Result[];
+};
+
+type Result = {
+  orderId: string;
+  bridgeName: string;
+  isCowswapTrade: boolean;
+  srcTransactionHash: string;
+  destTransactionHash: string;
+  fromChainId: number;
+  toChainId: number;
+  srcTxStatus: string;
+  destTxStatus: string;
+};
+
+/* 
+status: {
+  orderId: '0x0bfa5c44e95964a907d5f0d69ea65221e3a8fb1871e41aa3195e446c4ce855bbdaee4d2156de6fe6f7d50ca047136d758f96a6f067ee7474',
+  bridgeName: 'across',
+  isCowswapTrade: true,
+  fromChainId: 42161,
+  toChainId: 8453,
+  srcTransactionHash: '0x649b6fd231cf97972ccff205925f4582c760db7ce54d1b38a91eedca0e933986',
+  destTransactionHash: '0xe342cf2cb68ac161968457926f9449084777ca0662c94b88c2c783926552d189',
+  srcTxStatus: 'COMPLETED',
+  destTxStatus: 'COMPLETED'
+}
+*/
+
+export async function run() {
```

Comment:
Have added a script that shows three ways to fetch bridge status from our indexer based on cowswap order id